projects/resupply/abi/pairContract.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    totalCollateral: {
```

Comment:
we prefer human readable abi format as it is more compact, also, we have an internal format if function has only one output and no input, so this can be replaced with `uint256:totalCollateral`